buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -214,6 +214,42 @@
       }
     },
     {
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Packages -> Azure",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "msbuild",
+        "arguments": "publish.proj /p:CloudDropAccessToken=$(TransportFeedAccessToken) /p:CloudDropAccountName=$(TransportFeedAccountName) /p:ContainerName=$(Label) /p:__PublishPackages=true /p:OverwriteOnPublish=true /p:PackagesPattern=../packages/AzureTransfer/Release/pkg/*/*/*.nupkg /p:RelativePath=pkg /fileloggerparameters:Verbosity=diag;LogFile=publishpkg.log",
```

Comment:
@karajas @weshaggard is this the right RelativePath? Does it matter what the layout looks like inside of the transport feed container? Should the Container name be some specific format, rather than using the pipebuild label?

Reply:
No that is the correct relative path. See the other comments.